Multisig/UI/Safe Management/Create Safe/GoogleWeb3AuthLoginModel.swift

```diff
@@ -33,9 +34,63 @@ class GoogleWeb3AuthLoginModel {
                 await MainActor.run(body: { [weak self] in
                     guard let self = self else { return }
                     let key = data["privateKey"] as? String
+                    let publicAddress = data["publicAddress"] as? String
                     let userInfo = data["userInfo"] as? [String: Any] ?? [:]
                     let email = userInfo["email"] as? String ?? "email withheld"
 
+//                    if Web3AuthMFAService.mfaEnabled(userInfo: userInfo) {
```

Comment:
can this commented-out section be stashed?

Reply:
This was committed accidentally